ui/lib/replication/addon/components/replication-summary.js

```diff
@@ -66,10 +66,22 @@ export default Component.extend(ReplicationActions, DEFAULTS, {
     this.setProperties(DEFAULTS);
   },
 
+  transitionTo: computed('mode', 'replicationMode', function() {
+    return () => this.router.transitionTo('vault.cluster');
+  }),
+
   submit: task(function*() {
-    yield this.submitHandler(...arguments);
-    let wizard = this.get('wizard');
-    wizard.transitionFeatureMachine(wizard.get('featureState'), 'ENABLEREPLICATION');
+    let mode;
+    try {
+      mode = yield this.submitHandler.perform(...arguments);
+    } catch (e) {
+      // TODO handle error
+    }
+    // Take transitionTo outside of a yield because it unmounts the cluster and yield cannot return anything
```

Comment:
i'd move this comment above line 70 since it's describing why we need the custom `transitionTo` property